ee/dtr/architecture.md

```diff
@@ -89,16 +75,15 @@ DTR supports these storage backends:
 * OpenStack Swift
 * Microsoft Azure
 
-## How to interact with DTR
+## How to Interact with DTR
 
-DTR has a web UI where you can manage settings and user permissions.
+DTR offers a web UI for managing settings and user permissions.
 
 ![](images/architecture-4.svg)
 
-You can push and pull images using the standard Docker CLI client or other tools
-that can interact with a Docker registry.
+The standard Docker CLI client can be used to push and pull images, as can any other tool that is able to interact with a Docker registry.
```

Comment:
I think we can remove "standard", not sure what that means.

Reply:
Done.